test/message-stream.ts

```diff
@@ -156,20 +158,21 @@ describe('MessageStream', () => {
         objectMode: true,
         highWaterMark: 0,
       };
-
-      assert.deepStrictEqual(messageStream.options, expectedOptions);
+      // tslint:disable-next-line no-any
+      assert.deepStrictEqual((messageStream as any).options, expectedOptions);
     });
 
     it('should respect the highWaterMark option', () => {
       const highWaterMark = 3;
-      const ms = new MessageStream(subscriber, {highWaterMark});
+      const ms =
+          new MessageStream(subscriber as {} as Subscriber, {highWaterMark});
 
       const expectedOptions = {
         objectMode: true,
         highWaterMark,
       };
-
-      assert.deepStrictEqual(ms.options, expectedOptions);
+      // tslint:disable-next-line no-any
+      assert.deepStrictEqual((ms as any).options, expectedOptions);
```

Comment:
If you cast this as `FakePassThrough` you can avoid the any cast/comment.

Reply:
done thanks 👍